### DISCUSSION TITLE
Meeting Notes: Followup w/ Cometa team before their upcoming launch

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#182] [Meeting]
Meeting Attendees:
@AlgoRhythMatic  met w/ Nikita Gorokhov (Co-Founder & Backend Developer).

The purpose of this meeting was to discuss final questions leading into opening of Cometa's initial set of liquidity mining pools, and ensure that ASA Stats was prepared to participate in a way that is most beneficial for our community.

Questions from ASA Stats:
- Q: What types of pools will be available at launch?
     - A: LP mining (Farming) pools + distribution pools will be available at launch.  LaaS (Liquidity as a Service) pools will be available on testnet in September.  LaaS allows projects seeking liquidity to provide their tokens from one side, while users provide the other side of each pairing. All trading fees are distributed to users while the project eats the impermanent loss.

- Q: Are there any more details about potential for ALGO rewards (Aeneas-related)?  If so, how will these be measured or quantified for distribution per project?
     - A: the initial Tinyman LP mining pools that are paired w/ ALGO (such as ASASTATS/ALGO) that are defined for launch will receive matching ALGO rewards - up to a combined 5k ALGO equivalent value.

- Q: How are rewards calculated (daily/continuously/etc)?
     - A: Rewards are calculated continuously each block.

- Q: Could you please summarize pool locking and how are rewards claimed?
     - A: Locked pools prevent claiming for the duration of a defined lock timer.  After the lock timer expires, a user may claim rewards directly (at any time), or use a "Compound" button/UI to easily re-invest rewards.  The compound feature performs the multiple operations needed to sell/exchange rewards -> LP + re-stake.  In unlocked pools, rewards can be claimed or compounded at any time.  **Important note**: In locked pools, the lock timer resets after each subsequent claim.

- Q: Are projects able to add more rewards after the initial setup?
     - A: No current possibility to add more rewards after a pool is setup.
     
- Q: Are there any more details about any additional META rewards incentives?
     - A: No defined roadmap regarding META rewards/valuation at this time.

- Q: How long will projects have a zero fees?  Is future fee structure known?
     - A: Currently no fee structure.  Rough plans for a small flat fee up front + small % of rewards distribution.

Miscellaneous:
- Initial pools that wish to receive matching ALGO from Algorand Aeneas need to setup directly with Cometa via Nikita.
- There is a possibility to have entry and withdraw fees for each pool, however I indicated that we are likely not interested in this type of mechanism.
- There are likely no Aeneas rewards for an ASASTATS -> ASASTATS distribution pool.

**Comment by AlgoRhythMatic** [comment]
My personal thoughts after this followup discussion with Nikita from Cometa - I think we likely want two pools at Cometa's launch:
- Pool 1: Aeneas-incentivized ALGO/ASASTATS pool w/ 7-day locking
     - Rough estimate for cost = ~10m ASASTATS (~5k ALGO -OR- 1.65k USD equivalent) needed to qualify for full Aeneas matching reward.
     - Our current Tinyman LP pool value: ~30.37M ASASTATS + ~15.14k ALGO = ~9.97k USD
     - Assuming 100% of our existing Tinyman LP migrated to this pool:
          - 60-day pool = 166,666 ASASTATS + 83.33 ALGO rewards distributed per day = ~200% APR
          - 90-day pool = 111,111 ASASTATS + 55.55 ALGO rewards distributed per day = ~135% APR
          - 120-day pool = 83,333 ASASTATS + 41.66 ALGO rewards distributed per day = ~100% APR

- Pool 2: ASASTATS -> ASASTATS distribution pool w/ no locking
     - A no-fee (at the moment) replacement for Algostake/Yieldly distribution pools
     - Targeting an ~25% APR equivalent to replace Algostake may not so simple, as participation + pool duration will dictate APR (no fixed possibilities at the moment).
     - We do not need to launch this pool immediately, as there are no added Aeneas incentives.
     - Looking for thoughts as to how much to fund here given that adjustments cannot be made after pool is started.

Summary: I no longer believe it is beneficial to have various locked/unlocked LP farming/mining pools based on the discussion I had with Nikita.  I think it is preferable to have a single incentivized pool w/ 7-day locking.  I think we should fund w/ goal of receiving the full Aeneas matching incentive (my estimate is ~10m ASASTATS ), and that the attractive middle ground for APR is probably the 90-day pool duration.  The risk present for this pool is that there is low adoption that focuses high rewards for some initial period - though this is present w/ any LP mining reward incentive that has no cap.  For the distribution pool, I'd like feedback from @SCN9A, @ipaleka and others - to understand if we want to pursue this immediately, and how much to fund given that Cometa does not currently have the ability to apply a fixed APR rate.

**Comment by ipaleka** [comment]
I guess if no one complains, we should start with that 90 days pool you've suggested.

And fixed APR for ASASTATS-ASASTATS was introduced by our Algostake pool as there wasn't any other option, it's not that we insisted on a fixed APR. Please proceed with ASASTATS-ASASTATS in a way that we expect around 40-50M ASASTATS locked. I guess we can pick 120 days pool...

**Reply by AlgoRhythMatic** [reply]
> Please proceed with ASASTATS-ASASTATS in a way that we expect around 40-50M ASASTATS locked. I guess we can pick 120 days pool...

With a distribution pool like this, we are rewarding people that hold vs. increasing liquidity.  If we are simply looking to match/replace Algostake on a ~50m TVL for a 120-day pool, then ~4.1m rewards would suffice (25%).  I think this would make sense if we have an unlocked pool.  If we want to offer something more (say 50% or greater) - then I think we could also employ 7-day locking.  I personally focused on LP during last Yieldly round, so don't recall what the distribution pool's average APR was.  IMO, I think trying to incentive locked, stable liquidity is best use funds over the long term - but may be biased per my own investment style.

**Reply by wxivd** [reply]
> With a distribution pool like this, we are rewarding people that hold vs. increasing liquidity.

I think ASASTATS-ASASTATS staking rewards should not be more that liquidity rewards. Otherwise results may be counterproductive in terms of boosting ASA Stats liquidity

**Reply by ipaleka** [reply]
I guess it would be the safest pat for us the to create ASASTATS-ASASTATS pool for a minimum duration (60 days) to minimize risk of rewards being too high and compete with LP pool.

If they would be too low and everybody leaves we'd create a new one in parallel with bigger rewards.

**Reply by AlgoRhythMatic** [reply]
> I think ASASTATS-ASASTATS staking rewards should not be more that liquidity rewards. Otherwise results may be counterproductive in terms of boosting ASA Stats liquidity

100% agree.  The distribution pool is like a simple "thank you for holding" reward for existing community, whereas the liquidity incentive should be a motivator to attract additional investment -> LP.  Also - bigger risk, bigger reward.

> I guess it would be the safest pat for us the to create ASASTATS-ASASTATS pool for a minimum duration (60 days) to minimize risk of rewards being too high and compete with LP pool.

Seems reasonable to me.  If we are agreeing that we should try to target similar Algostake reward to treat Cometa distribution as a pure replacement, w/ a 60-day duration, I think we could target 2m ASASTATS.  If you wanted to increase slightly to ~35% APR (at 50m TVL pool), we could raise to 3m.

**Reply by AlgoRhythMatic** [reply]
> safest pat for us the to create ASASTATS-ASASTATS pool for a minimum duration

Oh - one more note here: minimum duration appears to be 10 days, so if we wanted to, we could even create a 30-day distribution pool w/ 1 or 1.5m ASASTATS as a "trial", and then update accordingly.

**Reply by ipaleka** [reply]
Oh, that's even better, please prepare everything then for 30 days pool and announce to Cometa's team that we're going to extend it after that initial period.

**Comment by AlgoRhythMatic** [comment]
Looks like start date is now planned for August 5th:
https://twitter.com/CometaHub/status/1551609049954885633
